Source/BugsnagNotifier.m

```diff
@@ -383,34 +388,15 @@ - (void)watchLifecycleEvents:(NSNotificationCenter *)center {
 }
 
 - (void)willEnterForeground:(id)sender {
-    [self.sessionTracker startNewSession:[NSDate date]
-                                withUser:self.configuration.currentUser
-                            autoCaptured:YES];
-
-    NSTimeInterval sessionTickSeconds = 60;
-
-    if (!self.sessionTimer) {
-        _sessionTimer = [NSTimer scheduledTimerWithTimeInterval:sessionTickSeconds
-                                                         target:self
-                                                       selector:@selector(sessionTick:)
-                                                       userInfo:nil
-                                                        repeats:YES];
-        [self sessionTick:self];
-    }
+    [self.sessionTracker handleAppForegroundEvent];
```

Comment:
Will this capture 2 sessions on application startup rather than 1? `startNewSessionIfAutoCaptureEnabled` will also start a session, unless I'm mistaken.

Reply:
It only captures 1 event, since the foreground session only happens when coming from the background and having `backgroundTime` more than 60s before.